test/simplifyExpression/distributeSearch/distributeSearch.test.js

```diff
@@ -47,6 +47,16 @@ describe('distribute', function () {
       ['(-2x^2 * 3x - 2x^2 * -4)',
         '(-6x^3 + 8x^2)']
     ],
+    ['(x - 2)^2',
+      ['(x * (x - 2) - 2 * (x - 2))',
```

Comment:
Ideally it might be nice if we had an intermediate step that was `(x-2)(x-2)` ... not sure if that's strictly necessary though!